README.md

```diff
@@ -95,7 +95,7 @@ You can build the local project `docker-compose` on Mac or Linux,  or into a dis
 
 1. `git clone git@github.com:mozilla-ai/lumigator.git`
 2. `make start-lumigator`
-3. The REST API should be available at http://localhost:8000. (If you need to change the port, you can do it in the[`docker-compose`](docker-compose) )
+3. The REST API should be available at http://localhost:8000. (If you need to change the port, you can do it in the[`docker-compose`](docker-compose.yaml) )
```

Comment:
```suggestion
3. The REST API should be available at http://localhost:8000. (If you need to change the port, you can do it in the [`docker-compose`](docker-compose.yaml) )
```

Thanks for this! While you're changing this line, would you mind also putting a space between `the` and `docker-compose`?